.github/release-initial-checklist.md

```diff
@@ -56,18 +56,22 @@ Additionally, make sure to differentiate between things in the testing notes tha
   * Note: the script automatically updates version numbers on Github (commits on your behalf).
   * **ALERT**: This script will ask you if this release will be deployed to WordPress.org. You should answer yes for this release even if it is a pre-release.
   * A GitHub release will automatically be created and this will trigger a workflow that automatically deploys the plugin to WordPress.org.
-  * An email confirmation is required before the new version will be released, so check your email in order to confirm the release.
-* [ ] Edit the [GitHub release](https://github.com/woocommerce/woocommerce-gutenberg-products-block/releases) and copy changelog into the release notes. Ensure there is a release with the correct version, the one you entered above.
-* [ ] The `#team-rubik` slack instance will be notified about the progress with the WordPress.org deploy. Watch for that. If anything goes wrong, an error will be reported and you can followup via the GitHub actions tab and the log for that workflow.
+  
 
-## After Workflow completes
+## If this release is deployed to WordPress.org...
 
+* [ ] An email confirmation is required before the new version will be released, so check your email in order to confirm the release.
```

Comment:
What do you think if we replace this string with the same version that there is on `patch-initial-checklist`?

```
An email confirmation is required before the new version will be released. Ask team members to check their email.
```

Reply:
@gigitux considering p1643799377147009-slack-C8X6Q7XQU I guess we can go ahead with the initial version?

Reply:
Yes, thanks for investigating this! 
Can you update `patch-initial-checklist.md` file, too?